File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/domene/virksomhetsklassifikasjoner/Næring.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.domene.virksomhetsklassifikasjoner;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Næring implements Virksomhetsklassifikasjon {
    private String kode;
    private String navn;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/domenemodeller/Næring.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.domenemodeller

data class Næring(
    // TODO: Rename til noe annet så vi skjønner at her får vi en næringskode
    // TODO: Legg til hent 2-siffer
    val kode: String,
    val navn: String
)